src/components/shared/Header.tsx

```diff
@@ -11,14 +11,14 @@ const headerItems = [
     tag: '#kral',
     text: 'Král Majálesu',
   },
-  // {
-  //   tag: '#partneri',
-  //   text: 'Partneři',
-  // },
   {
     tag: '#kontakt',
     text: 'Kontakt',
   },
+  {
+    tag: '#partneri',
+    text: 'Partneři',
+  },
```

Comment:
wouldn't be better to have link `partneri` before `kontakt`? 
on the website, the section `partner` is also before `kontakt` section (that's why)